### PR TITLE
[MOSIP-23218] Property updates from migrator.

### PR DESCRIPTION
**Pull request by Mohanraj209**

### CODE REVIEW

File: registration-processor-default.properties [hunk 1]
```diff
@@ -34,7 +34,7 @@ registration.processor.bio.dedupe.reprocess.buffer.time=900
 # Database hostname below is assuming postgres is running inside cluster in 'postgres' namespace
 # If database is external to production, provide the DNS or ip of the host and port 
 mosip.registration.processor.database.hostname=postgres-postgresql.postgres
-mosip.registration.processor.database.port=5432
+mosip.registration.processor.database.port=80
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: db is running in 80 ?

File: registration-processor-default.properties [hunk 2]
```diff
@@ -289,7 +289,7 @@ registration.processor.reprocess.limit=500
 registration.processor.pause.packets.for.backpressure=true
 # The reprocessor scheduler configurations
 # The elapse time (in sec) beyond which the rids will be considered for reprocessing
-registration.processor.reprocess.elapse.time=900
+registration.processor.reprocess.elapse.time=7200
```
Comment (vishwa-vyom) [inline registration-processor-default.properties]: why value is not match this ? https://github.com/mosip/mosip-config/blob/upgrade-v2/sandbox/registration-processor-mz.properties#L377

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 1 above)
Comment (Mohanraj209) [reply registration-processor-default.properties]: updated

File: registration-processor-default.properties (hunk 2 above)
Comment (Mohanraj209) [reply registration-processor-default.properties]: updated sir.

**Pull request closed without merging**